www/main/viewer/Slide/Slide.jsx

```diff
@@ -140,7 +141,7 @@ const Slide = ({ verseObj, nextLineObj, isMiscSlide }) => {
                 order={transliterationOrder}
               />
             )}
-            {displayNextLine && nextLineObj && (
+            {!akhandpatt && displayNextLine && nextLineObj && (
```

Comment:
@theamanjs veerji instead of hiding the next line here, can we disable it from setting.

Reply:
Veerji fixed